GeoDaCenter/chicago-environment-explorer
# feat: show min/max on map legend

## Problem
Min + Max values are not shown explicitly on the Legend

This is helpful information since the bins may not all be the same width

Fixes #130 

## Approach
* Calculate and show min + max values on the BinLabels

## How to Test
1. Navigate to https://deploy-preview-140--chicago-env-explorer.netlify.app/map
    * You should see that the bin values now extend to the edge of the Legend (at the bottom)
2. Select a few different variables
    * You should see the min + max values do not extend off of the Legend (worst case: Population Density)
